src/DynamoCoreWpf/Views/Preview/WatchTree.xaml

```diff
@@ -280,14 +250,24 @@
                   Background="White"
                   Opacity=".5"
                   BorderBrush="{x:Null}"
-                  Style="{StaticResource TreeView}"
                   VirtualizingStackPanel.IsVirtualizing="True"
                   VirtualizingStackPanel.VirtualizationMode="Recycling"
                   ItemContainerStyle="{StaticResource WatchTreeViewItem}"
                   SelectedItemChanged="TreeView1_OnSelectedItemChanged">
+
+            <TreeView.Style>
+                <Style TargetType="TreeView">
```

Comment:
Yes, in WPF it seems that even if you define the style within the context, you still have to set the `TargetType`. You can see the same example in `VirtualizingPanel` and `ListView.ItemContainerStyle`